ui/src/components/kv/KVTable.vue

```diff
@@ -543,14 +545,16 @@
             const type = kv.value.type;
             let value: any = kv.value.value;
 
-            if (["STRING", "DURATION", "JSON"].includes(type)) {
+            if (type === "STRING") {
+                value = `"${value}"`;
```

Comment:
We will have problems if the value contains a `"`

Should we prefer the more common?

```suggestion
                value = JSON.stringify(value)
```

Reply:
Nice spot my bad, ty, I pushed your suggestion.